packages/types/package.json

```diff
@@ -36,6 +36,16 @@
         "default": "./_cjs/ext.cjs"
       }
     },
+    "./Blog": {
```

Comment:
What are those key values for?

Reply:
ESM exports https://nodejs.org/api/packages.html#subpath-exports